src/server/router.js

```diff
@@ -0,0 +1,23 @@
+//-- Import Modules
+const fs = require('fs')
+const path = require('path');
```

Comment:
fs and path are unused in this file. I am happy to merge, but I think you should set up eslint to catch cases like this.

Reply:
it would also fix the indentation in this file!

Reply:
let me know when you've seen this comment or when you've fixed it, and I'll merge.

Reply:
we fixed it, and we will create a new branch to define eslint.